docs/userGuide/contentAuthoring.md

```diff
@@ -125,6 +125,29 @@ You can insert variables to be replaced by pre-defined values anywhere in your s
 
 Some helpful variables are provided by MarkBind for your convenience, as shown below. 
 
+#### Font Awesome Icons
+
+This asset is provided by [Font Awesome](https://fontawesome.com/) under their [free license](https://fontawesome.com/license).
+<br><br>

```

Comment:
I see that without the `<br><br>`, the code below is rendered literally.

I was able to avoid using `<br><br>` by specifying the code below as `<markdown>` explicitly. Not sure which one is a better option though:

```html
<tip-box>
Traditional usage:  
<markdown>`<span class='fas fa-chess-knight'></span>`</markdown>
<br>  
MarkBind usage:
<br>
<code>{<span></span>{fas_chess_knight}}</code>
</tip-box>
```

Reply:
The `<br>` tag between the 2 usage blocks is not needed. Looks cleaner when using the `<markdown>` tag.